docs/gno-infrastructure/validators/faq.md

```diff
@@ -107,7 +107,7 @@ either a full node or a pruned node, it is important to retain enough blocks to
 ### How do I generate `genesis.json`?
```

Comment:
This should be in gnogenesis/README.md, not in the validator's page.

Reply:
I honestly believe this is sort of out of the scope of this PR, but I've added a fresh README regardless:

e667e9c